ski_conditions/apps/app_scraping/tests/test_commands.py

```diff
@@ -0,0 +1,125 @@
+import os
+
+import pytest
+import responses
+from django.core import management
+
+from ..management.commands.do_scraping import (
+    HeavenlyScraper, HeavenlySnowReport, KirkwoodScraper, KirkwoodSnowReport, NorthstarScraper, NorthstarSnowReport
+)
+from ..models import SkiResort
+
+
+# pylint: disable=no-member
+@responses.activate
+@pytest.mark.django_db()
+def test_scraping():
+    '''
+    test lift and trail number scraping
```

Comment:
Similar comments as below. This comment is not useful. Also, how does this test differ from the one below?

DRY it up.

Reply:
Merged tests back into one. I separated them because of a pylint issue that said there were too many statements. But after replacing the duplicate statements with a loop, that issue was resolved.